next/pages/_app.tsx

```diff
@@ -63,7 +63,11 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
           <SSRProvider>
             <Component {...pageProps} />
 
-            {isClient && <DynamicChat />}
+            {isClient &&
+              pageProps.page.data?.length > 0 &&
+              (pageProps.page.data[0].id === '611' ||
+                pageProps.page.data[0].attributes.parentPage.data?.attributes.slug ===
+                  'bratislava-pre-ukrajinu') && <DynamicChat />}
```

Comment:
would prefer this pulled into `const shouldDisplayUkraineSupportChat = isClient && pageProps.page....` under the line where isClinet is defined